packages/seacas/applications/epu/EP_SystemInterface.C

```diff
@@ -507,6 +501,19 @@ bool Excn::SystemInterface::parse_options(int argc, char **argv)
   else {
     throw std::runtime_error("\nERROR: (EPU) basename not specified\n");
   }
+
+  // Check that subcycle count does not match processor count --
+  // in that case the existing files will be overwritten.
+  if (processorCount_ <= subcycle_) {
+    if (myRank_ == 0) {
+      fmt::print(stderr,
```

Comment:
This is going into the C++ Standard Library soon! :-D

Reply:
Yes!  I was glad to hear that.  Although I may be gone by the time we are allowed to use C++20...

Reply:
@gsjaardema It may not be quite so long -- we're thinking about C++14 finally.  C++17 is a bigger jump due to semantics changes, but 20 might not be so bad.